Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -2613,6 +2613,46 @@ def test_empty_readable_outputs(self):
         context = res.to_context()
         assert "outputs_test" == context.get('HumanReadable')
 
+    def test_replace_existing(self):
+        """
+        Given:
+        - replace_existing=True
+
+        When:
+        - Returning an object to context that needs to override it's key on each run.
+
+        Then:
+        - Return an object with the DT "(true)"
+        """
+        from CommonServerPython import CommandResults
+        res = CommandResults(
+            outputs="next_token",
+            outputs_prefix="Path.To.Value",
+            replace_existing=True
+        )
+        context = res.to_context()
+        assert context["EntryContext"] == {"Path.To(true)": {"Value": "next_token"}}
+    
+    def test_replace_existing_not_nested(self):
+        """
+        Given:
+        - replace_existing=True but outputs_prefix is not nested, i.e., does not have a period.
+
+        When:
+        - Returning an object to context that needs to override it's key on each run.
+
+        Then:
+        - Raise an errror.
+        """
+        from CommonServerPython import CommandResults
+        res = CommandResults(
+            outputs="next_token",
+            outputs_prefix="PathToValue",
+            replace_existing=True
+        )
+        with pytest.raises(DemistoException, match='outputs_prefix must be a nested path to replace an existing key.'):
```

Comment:
need to assert false after tocontext, otherwise even if exception doesnt happen test will pass

Reply:
I got this message when I added a dot to the path so that an error was not raised:
`Failed: DID NOT RAISE <class 'CommonServerPython.DemistoException'>`